spec/active_record/comments_spec.rb

```diff
@@ -72,6 +72,26 @@ def capture_sql
       end
       sql.should == 'SELECT * FROM "users" WHERE "users"."id" = 1 /* xxx */'
     end
+
+    it "should be thread safe" do
+      res = []
+      ["xx", "yy"].map do |comment|
```

Comment:
ahhh I misread that the sleep makes sure both get started :)
(from my experience 0.1 is good enough)